### PR TITLE
Add title to to-top button

### PR DESCRIPTION
**Pull request by Stanzilla**

### CODE REVIEW

File: material/partials/languages/en.html [hunk 1]
```diff
@@ -33,5 +33,6 @@
   "source.file.date.updated": "Last update",
   "source.file.date.created": "Created",
   "tabs.title": "Tabs",
-  "toc.title": "Table of contents"
+  "toc.title": "Table of contents",
+  "topbutton.title": "Back to Top"
```
Comment (squidfunk) [inline material/partials/languages/en.html:37]: Thanks! Could you rename it into `top.title`? I know that's probably a little less self-explanatory, but it matches the style of `toc` and `tabs` with their feature flags:

- `tabs` -> `navigation.tabs`
- `toc` -> `navigation.toc`
- `top` -> `navigation.top`

I'm all for keeping consistency.

**Review by squidfunk — CHANGES_REQUESTED** [review]

File: material/partials/languages/en.html (hunk 1 above)
Comment (Stanzilla) [reply material/partials/languages/en.html:37]: Yup, done!

Comment (squidfunk) [inline material/partials/languages/en.html:37]: One more missing 😅

**Review by squidfunk — CHANGES_REQUESTED** [review]

File: src/partials/languages/en.html [hunk 2]
```diff
@@ -53,5 +53,6 @@
   "source.file.date.updated": "Last update",
   "source.file.date.created": "Created",
   "tabs.title": "Tabs",
-  "toc.title": "Table of contents"
+  "toc.title": "Table of contents",
+  "top.title": "Back to Top"
```
Comment (squidfunk) [inline src/partials/languages/en.html:57]: Sorry, one more thing - I've overseen. It should be "Back to top", not "Back to Top".

**Review by squidfunk — CHANGES_REQUESTED** [review]

**Comment by squidfunk** [comment]
Code review:

10 lines of code – "hmm that variable isn't named appropriately", "you're missing indentation", "no we should indent like this to congey meaning"

500 lines of code – APPROVED. next.

**Pull request merged**